### DISCUSSION TITLE
Is there a mechanism for selecting iframes?

### DISCUSSION THREAD

**Discussion by ethan-leba** [qutebrowser/qutebrowser#6316] [Q&A] [answered]
Hi all,

Sorry if this is already in the documentation but I wasn't able to find any sort of keybinding or command for selecting an iframe. Is there any built-in support for this already? Some websites have the main scrollable content of a website in an iframe, so I'd like to be able to focus it, but I've been having to click on it with my mouse so far.

Example of a website that I'm having this issue with: https://www.cs.utexas.edu/users/moore/acl2/manuals/current/manual/index-seo.php/ACL2____INTRODUCTION-TO-REWRITE-RULES-PART-1

Thanks!

**Comment by The-Compiler** [comment, marked as answer]
There are no (i)frames on that page, it's just multiple scrollable div's. Right now there isn't a way to select them without using the mouse, see #4715.

**Reply by bitraid** [reply]
Unless you press <kbd>tab</kbd> / <kbd>shift-tab</kbd> a few times to highlight a link first.
